README.md

```diff
@@ -25,32 +25,40 @@ buildscript {
 plugins {
   id "net.ltgt.apt" version "0.11"
   id "net.ltgt.errorprone" version "0.0.11"
-  id "java"
+  id "java-library" // Or just plain old "java"
 }
 
 dependencies {
     apt "com.uber.nullaway:nullaway:0.1.2"
 
-    compile "com.google.code.findbugs:jsr305:3.0.2"
+    // Optional, some source of nullability annotations
+    compileOnly "com.google.code.findbugs:jsr305:3.0.2"
```

Comment:
CompileOnly is not compatible with the `tasks.withType(JavaCompile)` bit as testCompileOnly does not extend from compileonly

Reply:
What do you suggest instead? I don't think recommending people compile this is the right approach

Reply:
Here I think we are fine assuming we don't suggest running NullAway on test code by default.

Reply:
`compileOnly` is fine here